src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -135,9 +135,15 @@ Object.assign(SyntheticEvent.prototype, {
 
     if (event.stopPropagation) {
       event.stopPropagation();
-    } else {
+    } else if (typeof event.cancelBubble !== 'unknown') { // eslint-disable-line valid-typeof
```

Comment:
Technically no, though it's against the React style guide all the same. 

ESLint seems to struggle with `else if`. The only other valid option I can manage is:

```
if (event.stopPropagation) {
  event.stopPropagation();

  /* eslint-disable valid-typeof */
} else if (typeof event.cancelBubble !== 'unknown') {
  /* eslint-enable valid-typeof */
}
```

Or we could flip the if to place `typeof event.cancelBubble` before the `event.stopPropagation` check.

I know this is annoying and silly. Thank you for your patience on this.